app/src/main/java/com/example/photogallery/Presenters/GalleryPresenter.java

```diff
@@ -53,13 +56,18 @@
     private double longitude = 0.0;
 
     private IPhotoRepository photoRepository;
+    private IFileHelper fileHelper;
+
     private Activity context;
     private List<Photo> photos;
     private int index = 0;
 
     public GalleryPresenter(Activity context) {
         this.context = context;
+
```

Comment:
It is best practice to try to avoid adding unnecessary whitespace when you are committing  changes.  It starts to add extra noise for reviewers when they are going over your PR.